.github/workflows/package-manager-ci.yml

```
@@ -0,0 +1,55 @@
+name: package-manager-ci
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  pnpm:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ${{ matrix.os }}
```

Otherwise you'll only be running these on `ubuntu-latest` instead of the matrix specified `ubuntu-16.04`.